modules/tile-converter/test/3d-tiles-converter/3d-tiles-converter.spec.js

```diff
@@ -7,7 +7,7 @@
 //   'https://tiles.arcgis.com/tiles/u0sSNqDXr7puKJrF/arcgis/rest/services/Frankfurt2017_v17/SceneServer/layers/0';
 // const PGM_FILE_PATH = '@loaders.gl/tile-converter/test/data/egm84-30.pgm';
 
-// test('cli - Converters#converts i3s to 3d-tiles tileset', async t => {
+// test('tile-converter - Converters#converts i3s to 3d-tiles tileset', async t => {
```

Comment:
Might be good to cut unused code, or add comments explaining why we keep it.

Reply:
Right. It's right in the 1st line of the file